### PR TITLE
Update OnlineSBI.com.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

File: src/chrome/content/rules/OnlineSBI.com.xml [hunk 1]
```diff
@@ -44,14 +40,11 @@ Fetch error: http://cmp.onlinesbi.com/ => https://cmp.onlinesbi.com/: (28, 'Conn
...
 -->
-<ruleset name="OnlineSBI.com" default_off='failed ruleset test'>
-
-	<!--	Direct rewrites:
-				-->
+<ruleset name="OnlineSBI.com">
+	<target host="onlinesbi.com" />
+	<target host="www.onlinesbi.com" />
 	<target host="acs.onlinesbi.com" />
-	<target host="cmp.onlinesbi.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/OnlineSBI.com.xml]: Please add a comment for it

Comment (koops76) [inline src/chrome/content/rules/OnlineSBI.com.xml]: @J0WI Done.

File: src/chrome/content/rules/OnlineSBI.com.xml [hunk 2]
```diff
@@ -1,46 +1,14 @@
...
-		- mpi
-		- oaa
-		- pg
-		- prepaid
-		- remit
-		- retail
-
-	Nonfunctional subdomains:
-
-		- cms	(self-signed)
```
Comment (J0WI) [inline src/chrome/content/rules/OnlineSBI.com.xml]: Please keep a target/comment for this host

Comment (koops76) [inline src/chrome/content/rules/OnlineSBI.com.xml]: @J0WI Is it not self-signed?

Comment (J0WI) [inline src/chrome/content/rules/OnlineSBI.com.xml]: `https://cms.onlinesbi.com/CMS/` WFM

Comment (koops76) [inline src/chrome/content/rules/OnlineSBI.com.xml]: @J0WI Done.

**Pull request merged**